gravitee-apim-console-webui/src/management/api/apis.route.ts

```diff
@@ -1481,7 +1482,7 @@ export const states: Ng2StateDeclaration[] = [
   },
   {
     name: 'management.apis.runtimeLogs-messages',
-    url: '/runtime-logs/:requestId',
+    url: '/runtime-logs/message/:requestId',
```

Comment:
'/runtime-logs/messages/:requestId'

Reply:
but in my opinion it weird to have two different routes depending on the type of the api
should we simply keep a `management.apis.runtimeLogs` route which would be responsible of displaying the correct sub components ?

Reply:
Done 👍🏼